LowerH: 36
LowerS: 25
LowerV: 25
UpperH: 86
UpperS: 255
UpperV: 255
qos_overrides./parameter_events.publisher.depth: 1000
qos_overrides./parameter_events.publisher.durability: volatile
qos_overrides./parameter_events.publisher.history: keep_last
qos_overrides./parameter_events.publisher.reliability: reliable
use_sim_time: false
